.github/workflows/ci.yml

```diff
@@ -0,0 +1,96 @@
+name: GitHub CI
+
+on:
+  push:
+    paths-ignore:
+      - '.github/*TEMPLATE/**'
```

Comment:
```suggestion
      - '*.md'
      - '.github/*'
      - '.github/*TEMPLATE/**'
```

Reply:
Any markdown changes should be pushed into artifacts, in case of important documentation.